pylib/Tools/Harasser/Harasser.py

```diff
@@ -85,7 +85,7 @@ def get_running_harassers(self):
     def parallel_execute(self, cmds, cmdargs, testDef):
         """This function is passed into multiprocessing as a target
         """
-        status,stdout,stderr,time = testDef.execmd.execute(cmds, cmdargs, testDef)
+        status,stdout,stderr,time,_ = testDef.execmd.execute(cmds, cmdargs, testDef)
```

Comment:
I'm not a python expert, but is this correct?  the return value isn't being treated as a dictionary like in other places in the PR.

Reply:
Garrrr - missed that one. Fixed it. Thanks!